packetbeat/main.go

```diff
@@ -5,6 +5,17 @@ import (
 
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/packetbeat/beater"
+
+	// import support protocol modules
+	_ "github.com/elastic/beats/packetbeat/protos/amqp"
```

Comment:
not sure I'd want to full automate this. nice thing about having imports in main.go only is, one can import a very basic packetbeat as is, without any modules (for example to have a flowsbeat connection only flow information).